pr2eus_tutorials/launch/pr2_tabletop.launch

```diff
@@ -3,30 +3,47 @@
   <arg name="publish_objects_tf" default="true" />
   <arg name="launch_tracking" default="false" />
   <arg name="manager" default="pr2_tabletop_object_detector_nodelet_manager" />
+  <arg name="sim" default="false" />
   <arg name="remote" default="false" />
-  <arg name="load_machine" default="false" unless="$(arg remote)" />
-  <arg name="load_machine" default="true" if="$(arg remote)" />
-  <arg name="machine" default="c2" unless="$(arg remote)" />
-  <arg name="machine" default="localhost" if="$(arg remote)" />
-  <arg name="camera_name" default="kinect_head" />
+  <arg name="load_machine" default="true" />
 
-  <arg name="RGB_CAMERA_INFO" value="/$(arg camera_name)/rgb/camera_info" />
-  <arg name="RGB_IMAGE" value="/$(arg camera_name)/rgb/throttled/image_rect_color" />
-  <arg name="RGB_IMAGE_REMOTE" value="/$(arg camera_name)_remote/rgb/image_rect_color" />
-  <arg name="DEPTH_CAMERA_INFO" value="/$(arg camera_name)/depth_registered/camera_info" />
-  <arg name="DEPTH_IMAGE" value="/$(arg camera_name)/depth_registered/throttled/image_rect" />
-  <arg name="DEPTH_IMAGE_REMOTE" value="/$(arg camera_name)_remote/depth_registered/image_rect" />
-  <arg name="PROSILICA_CAMERA_INFO" value="/prosilica/camera_info" />
-  <arg name="PROSILICA_IMAGE" value="/prosilica/image_raw" /> 
-  <arg name="PROSILICA_IMAGE_REMOTE" value="/prosilica_remote/image_raw" /> 
-  <arg name="POINT_CLOUD" value="/$(arg camera_name)/depth_registered/throttled/points" />
-  <arg name="POINT_CLOUD_REMOTE" value="/$(arg camera_name)_remote/depth_registered/points" />
+  <arg name="robot" value="$(optenv ROBOT sim)" if="$(eval arg('remote') or arg('sim'))" /> 
```

Comment:
You use `eval` arg, but `eval` can be used in kinetic or newer.
http://wiki.ros.org/roslaunch/XML

Our PR2 is indigo, so I think we cannot use this launch file with SSH mode.
(I am sorry if my understanding is wrong)

Reply:
that's true.

Reply:
The solution I only come up with is use `if` arg or `unless` arg instead of `eval` arg?

Reply:
> The solution I only come up with is use if arg or unless arg instead of eval arg?

I don't have any other ideas either....

By the way, do we really need `load_machine` arg?
If we need to use `load_machine`, please update README.md.
```
  <include file="$(find pr2_machine)/$(arg robot).machine" if="$(arg load_machine)" />
```

Reply:
Sorry to be late. 

> By the way, do we really need load_machine arg?

No need anymore. I deleted it by https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/eabc88c49269a2a50b6bc772a12ba012b88777b2.

Reply:
> You use eval arg, but eval can be used in kinetic or newer.

I change `if` arg or `unless` arg instead of `eval` arg by https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/02de16a37fcd34c15a603c29d04dbc9bc907dfe4 .

Reply:
Sorry to be late. 

> By the way, do we really need load_machine arg?

No need anymore. I deleted it by https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/557aa9b2d63ccd710ae244dc5a4072389ead1cad

Reply:
> You use eval arg, but eval can be used in kinetic or newer.

I change `if` arg or `unless` arg instead of `eval` arg by https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/5687bb50a46a4954d9f47aa09f4c2b9a73452091

Reply:
I've never seen this writing style before, but it's interesting.

I hope this PR works well.

Reply:
Yes, I confirmed it works well with gazebo mode, ssh mode and remote mode.